data/data.go

```diff
@@ -0,0 +1,422 @@
+// Package data implements sql operations over the NoName e.V. website-db.
+package data
+
+import (
+	"database/sql"
+	"errors"
+	"flag"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var (
+	driver  = flag.String("driver", "postgres", "Der benutzte sql-Treiber")
+	connect = flag.String("connect", "dbname=nnev user=anon host=/var/run/postgresql sslmode=disable", "Die Verbindusgsspezifikation")
```

Comment:
ok, automated comment-formatting is reasonable. just seemed strange having this one long line, but comments splittet after 75 chars.